src/lib/menu/menu-trigger.ts

```diff
@@ -243,11 +243,18 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
       menu._resetAnimation();
 
       if (menu.lazyContent) {
+        menu.lazyContent._attached.pipe(take(1)).subscribe(() => console.log('re-attached'));
```

Comment:
Debugging statement?

Reply:
ops, I forgot about that one. It's removed.